josue/js/app.js

```diff
@@ -0,0 +1,52 @@
+import FormValidator from './Formvalidator.js';
+
+const form = document.querySelector('.form');
+const signInBtn = document.querySelector("#signBtn");
+const registerBtn = document.querySelector("#registerBtn");
+const submitBtn = document.querySelector("#submit");
+const registerFields = document.querySelector("#register");
+let fields = ['name','phone'];
+
+registerFields.remove();
+signInBtn.addEventListener("click",e=>{
+    registerFields.classList.add("register--hidden");
+    registerFields.remove();
+    submitBtn.innerText = "Sign In"
+    fields = ['name','phone']
+    validator.updateFields(fields,"signin")
+
+})
+registerBtn.addEventListener("click",e=>{
+    form.insertBefore(registerFields,submitBtn);
+    registerFields.classList.remove("register--hidden");
+    submitBtn.innerText = "Register"
+    const newFields= ['email','password','password_confirmation']
+    validator.updateFields(newFields,"register")
+});
+submitBtn.addEventListener('click',e=>{
+    console.log("click");
+    e.preventDefault();
+    if(validator.validateOnSubmit()){
+        // Fields are filled correctly now we check if user wants to register or login
+        if(submitBtn.innerText == 'Sign In'){
```

Comment:
NIT: Is good practice to store all strings in a variable at the top so if something changes is easier to locate the string at the top of the js file instead of going through all the places we are making reference to it, so you can do `const SIGN_IN = 'Sign In'`